Modelica/Electrical/Polyphase/Sensors/AronSensor.mo

```diff
@@ -75,9 +75,10 @@ Contains two
 to measure total active power in a three-phase system.
 </p>
 <p>
-This device works only for sinusoidal voltages and currents in three-phase systems without neutral.
+This device works only in three-phase systems without neutral.
 </p>
 <p>
+Massimo Ceraolo: Electric Power Systems - University of PISA <br>
 Elmar Schr&uuml;fer, Leonhard Reindl, Bernhard Zagar: Elektrische Messtechnik. Carl Hanser Verlag.<br>
 Reinhard Lerch: Elektrische Messtechnik. Springer Vieweg.
```

Comment:
I am unhappy with this literature, as these reference are not in line with the guidelines of the Modelica Standard Library, see `Modelica.UsersGuide.Conventions.Documentation.Format.References`. As we are lacking a User's Guide in the `Analog` package, which shall summarize all references, we are more and more getting inconsistent here.

Reply:
On line 81 I want you to add a http(s) link to the original PDF document or the publisher where the document is available so that one does not have to search on the internet for the cited literature. I will also add the links for the two references in line 82-83.

Reply:
Line 81 is totally wrong. My mistake. 
In fact, I did not even understand that the bottom lines should be references (probably because the two existing ones they are incomplete; the German language did not help).

Personally I'm not sure the Aron connection really requires reference(s). The very name should recall users the meaning. 
I also don't like references in languages different from English, for general topics for which there exists large enough English literature.

What I propose now is one of the following options:
1. remove all references and don't substitute them with anything (i.e. stop the documentation with the word "neutral.")
2. remove all references and substitute them with a unique reference in English language, possibly openly available to anyone on the Internet
3. remove all references and substitute them with some formulas and a picture that, starting from Tellegen's theorem, illustrates the reasons why Aron connection works not only "in three-phase systems without neutral" as is already mentioned in the documentation for easy comprehension, but, more in general "in all circuits composed by two subcircuits connected to each other by three wires, typically three-phase systems without neutral".

I personally propose option 1, but in case there is agreement on option 3, I could prepare it myself and create a different PR.

Reply:
Different Modelica users have a different level of knowledge on electrical engineering so I suppose it is good to have some explanation or reference -- either one is fine with me. I agree that it is a lot better to have an English reference instead of a German one. However, if literature references are used, the models shall only refer to the reference by reference key which points to `References`. All `References` with are summarized the `User's Guide` (see `Modelica.UsersGuide.Conventions.Documentation.Format.References`) and shall have a http(s) link to the real book, paper, thesis, etc.

I am therefore in favor of option 2 and 3. I think we should also include @AHaumer's opinion here before creating an new PR. 

@max-privato thanks for taking care of this issue.

Reply:
Users of Electrical|MultiPhase Library have PowerSensor and AronSensor available.

@christiankral from your words I understand that you expect anyone to be at ease with PowerSensor, while more explanation is needed in AronSensor. According to this idea, I could, after @AHaumer opinion, open a new PR with this additional info to be added to Aron Sensor's. In that case, this PR should be taken as invalid.

I could add this just after "This device works only in three-phase systems without neutral."